### PR TITLE
Use a different icon in macOS dark mode

### LINKED ISSUE #1640: macOS dark mode icon
Would be a nice thing to have

### PR DESCRIPTION
**Pull request by SubJunk**
Closes #1640

### CODE REVIEW

**Comment by js-kyle** [comment]
@SubJunk Could you look at the merge conflict? Otherwise I can

**Comment by js-kyle** [comment]
Works well on MacOs Catalina

File: src/main/java/net/pms/io/BasicSystemUtils.java [hunk 1]
```diff
@@ -233,6 +234,27 @@ public int getPingPacketFragments(int packetSize) {
...
+	 * @return whether the computer is running in dark mode
+	 */
+	private boolean isDarkMode() {
+		if (!Platform.isMac()) {
+			return false;
+		}
+
+		try {
+			// check for exit status only. Once there are more modes than "dark" and "default", we might need to analyze string contents..
+			final Process proc = Runtime.getRuntime().exec(new String[]{"defaults", "read", "-g", "AppleInterfaceStyle"});
```
Comment (js-kyle) [inline src/main/java/net/pms/io/BasicSystemUtils.java]: Personal nitpick:
```suggestion
                         String cmd = "defaults read -g AppleInterfaceStyle";
			final Process proc = Runtime.getRuntime().exec(cmd);
```

Comment (Nadahar) [reply src/main/java/net/pms/io/BasicSystemUtils.java]: @js-kyle The only problem is that the result isn't the same :wink: The working code places each argument as an element in a string array, your suggestion creates one space-separated string.

**Pull request merged**